packages/coil-extension/manifest.json

```diff
@@ -32,7 +32,7 @@
       "run_at": "document_start"
     }
   ],
-  "permissions": [ "webNavigation", "<all_urls>" ],
+  "permissions": [ "webNavigation", "<all_urls>", "tabs" ],
```

Comment:
Though mostly likely the latter, I am not sure if it's simply the addition of the `tabs` permission or the different token retrieval method that is the "fix" here

Reply:
It actually may be the case that it *IS* simply the tabs permission:
https://github.com/coilhq/web-monetization-projects/issues/2048#issuecomment-1065740650

Which call?